stories/chats/src/BubbleQRCode.tsx

```diff
@@ -96,6 +99,14 @@ export const BubbleQRCode: React.FC<QRCodeMessage> = (message: QRCodeMessage) =>
         UIConstant.mediumBorderRadius(),
     );
     const styles = useStyles(theme);
+    const ref = React.useRef<QRCodeRef>(null);
+    const onPress = React.useCallback(async () => {
+        const base64 = await ref.current?.getPng();
+        if (!base64 || !message.onPress) {
+            return;
+        }
+        message.onPress(base64);
+    }, []);
```

Comment:
Otherwise you'll be getting a png on any press even if the prop is not provided
```suggestion
    const onPress = React.useCallback(async () => {
        if (!message.onPress) {
            return;
        }
        
        const base64 = await ref.current?.getPng();
        if (!base64) {
            return;
        }

        message.onPress(base64);
    }, []);
```